python/cudf/pyproject.toml

```
@@ -174,3 +130,17 @@ wheel.packages = ["cudf"]
 provider = "scikit_build_core.metadata.regex"
 input = "cudf/VERSION"
 regex = "(?P<value>.*)"
+
+[tool.ruff.lint.isort]
```

For all these changes, we need to also add a section:

```suggestion
[tool.ruff]
extend = "../../pyproject.toml"

[tool.ruff.lint.isort]
```

So that we pick up the global configration as well.